src/Bridge/Doctrine/Orm/Filter/AbstractFilter.php

```diff
@@ -78,125 +77,35 @@ public function apply(QueryBuilder $queryBuilder, QueryNameGeneratorInterface $q
     abstract protected function filterProperty(string $property, $value, QueryBuilder $queryBuilder, QueryNameGeneratorInterface $queryNameGenerator, string $resourceClass, string $operationName = null/*, array $context = []*/);
 
     /**
-     * Gets class metadata for the given resource.
-     */
-    protected function getClassMetadata(string $resourceClass): ClassMetadata
```

Comment:
There seem to be a BC break here, because this method is missing from the OrmPropertyHelperTrait

Reply:
Fixed.